src/app/core/services/basic-rundown-state.service.ts

```diff
@@ -131,7 +130,7 @@ export class BasicRundownStateService implements OnDestroy {
     this.eventSubscriptions.forEach(eventSubscription => eventSubscription.unsubscribe())
   }
 
-  public getNonIdleRundown(): BasicRundown | undefined {
-    return this.basicRundowns.find(rundown => [RundownMode.ACTIVE, RundownMode.REHEARSAL].includes(rundown.mode))
+  public getBasicRundowns(): BasicRundown[] {
```

Comment:
We already have a method in this file that basically does the same as this one: `subscribeToBasicRundowns()`. We should ideally only have one method that gives us the BasicRundowns.

Reply:
I think there is a great enough of difference between getting a snapshot of the state and subscribing to the state to have this method.